content/scripts/build.js

```diff
@@ -783,7 +783,7 @@ class Builder {
       // or gets these references wrong.
 
       if (parentData) {
-        if (!parentData.hasOwnProperty("translations")) {
+        if (!("translations" in parentData)) {
```

Comment:
I suppose `obj.hasOwnProperty(prop)` is frowned upon now because of the possibility that an object was created with `Object.create(null)`, which makes sense, and I think the recommended approach is to use `Object.prototype.hasOwnProperty.call(obj, prop)` instead. In practice, we know when the property isn't within the prototype chain, and the form `prop in obj` is so much nicer to read, so I'm OK with this.

The purist in me thinks that maybe we should consider creating and using a convenience function, something like:
```js
function hasOwnProperty(obj, prop) {
  return Object.prototype.hasOwnProperty.call(obj, prop);
}
```

Reply:
I realize my mistake now. Ugh! I buckled at the eslint recommendation of not using `hasOwnProperty` and instead of RTFM I switch it to the `in` operator and the eslint errors went away. So it should have been `Object.prototype.hasOwnProperty.call` which is stupidly long. I miss Python. 

The crazy thing is that we're not writing a lib so we actually are very intimate with what our object keys are. 
The documentation says *"You may want to turn this rule off if your code only touches objects with hardcoded keys"* which is exactly us. We're doing `parentData.hasOwnProperty("translations")` not `parentData.hasOwnProperty(request.query.get('key'))`.
But now I see that that logic is equally applicable to the `in` operator. It works because it's a hardcoded string and it will always work. And it will work equally with `in` or with `hasOwnProperty`. 
The "dangerous" examples mentioned in https://masteringjs.io/tutorials/fundamentals/hasownproperty don't apply to our use. 

So, `hasOwnProperty` was fine. Strange that `eslint` thought it was more OK to use `in`. The most correct one `Object.prototype.hasOwnProperty.call` is so long that it hurts my eyes. Having to write a convenience function seems plausible but in retrospect overkill.

I fell into this trap. What do you want to do? Just merge it or just kill it or rewrite it?

Reply:
@peterbe As I noted above with `"In practice, we know when the property isn't within the prototype chain"` and as you said, I think even better, with `"we're not writing a lib so we actually are very intimate with what our object keys are."`, I'm OK with our use of `prop in obj` (and especially since the keys are hardcoded). Let's merge it! 🚀